script/deployers/MumbaiMangroveFullTestnetDeployer.s.sol

```diff
@@ -29,16 +33,25 @@ import {MgvReader} from "mgv_src/periphery/MgvReader.sol";
  * - open markets: DAI/USDC, WETH/DAI, WETH/USDC
  */
 contract MumbaiMangroveFullTestnetDeployer is Deployer {
+  uint internal maticPrice;
+
   function run() public {
     runWithChainSpecificParams();
     outputDeployment();
   }
 
+  function toGweiOfMatic(uint price) internal view returns (uint) {
+    return (price * 10 ** 9) / maticPrice;
+  }
+
   function runWithChainSpecificParams() public {
     // Deploy Mangrove
     new MumbaiMangroveDeployer().runWithChainSpecificParams();
     Mangrove mgv = Mangrove(fork.get("Mangrove"));
     MgvReader reader = MgvReader(fork.get("MgvReader"));
+    IPriceOracleGetter priceOracle =
+      IPriceOracleGetter(IPoolAddressesProvider(fork.get("Aave")).getAddress("PRICE_ORACLE"));
```

Comment:
Are we fairly confident on the prices aave provide? but I see we only use it for Mumbai right now, so that probably makes it ok.

Reply:
Yes we can be confident. Actually the prices on mumbai are fixed (1 USD for stables and 4000 for ether)